src/Smalot/PdfParser/Font.php

```diff
@@ -109,7 +115,9 @@ public function translateChar($char, $use_default = true)
      */
     public static function uchr($code)
     {
-        return html_entity_decode('&#'.((int) $code).';', ENT_NOQUOTES, 'UTF-8');
+        // html_entity_decode() will not work with UTF-16 or UTF-32 char entities,
+        // therefore, we use mb_convert_encoding() instead
+        return  mb_convert_encoding('&#'.((int) $code).';', 'UTF-8', 'HTML-ENTITIES');
```

Comment:
Super tiny thing: can you remove that second white space after the `return`?

Reply:
Will be fixed with the next commit. Thos looks like something the cs-fixer should catch, doesn't it?

Reply:
It should, but I don't know which rule to enforce here.